agate-core/src/main/java/org/obiba/agate/web/model/RealmConfigDtos.java

```diff
@@ -45,7 +45,8 @@ public RealmConfigDtos(LocalizedStringDtos localizedStringDtos, ConfigurationSer
       .setStatus(Agate.RealmStatus.valueOf(config.getStatus().toString()))
       .addAllGroups(config.getGroups())
       .setContent(ensureSecuredContent(configurationService.decrypt(config.getContent()), config.getType()))
-      .setUserCount(userRepository.countByRealm(config.getName()));
+      .setUserCount(userRepository.countByRealm(config.getName()))
+      .setUsernameClaim(config.getUsernameClaim());
```

Comment:
I think it will fail if value is null. Please verify.

Reply:
checked here https://github.com/obiba/obiba-commons/blob/0a4d7ededbcc34389f5b2b47fc0f05a5b02c0e7c/obiba-oidc/src/main/java/org/obiba/oidc/OIDCCredentials.java#L125-L142

Reply:
`uname` will be null